public/docs/_examples/quickstart/ts/app/app.module.ts

```diff
@@ -1,13 +1,16 @@
 // #docregion
-import { NgModule }      from '@angular/core';
-import { BrowserModule } from '@angular/platform-browser';
+import { Component, NgModule } from '@angular/core';
+import { BrowserModule }       from '@angular/platform-browser';
 
-import { AppComponent }   from './app.component';
+@Component({
+  template: `<h1>Hello Angular!</h1>`,
+  selector: 'my-app'
+})
+export class AppComponent { }
```

Comment:
On the one hand, exporting this here for a hello world can be confusing but on the other hand, removing it could confuse users as well.